core/includes/database/database.inc

```diff
@@ -549,7 +576,7 @@ abstract class DatabaseConnection {
   /**
    * Tells this connection object what its key is.
    *
-   * @param $target
+   * @param string $key
```

Comment:
The function's code attempts to handle that it's not set, so that might be null at some point? Not sure.